src/communication/serverMessages.ts

```diff
@@ -25,16 +26,18 @@ export interface Data {
 
 export interface CommandData {
     Command: string;
-    JsonMessage: string;
+    Data: any;
 }
 
 export let convertToWebsocketInputFormat = (data: Data): string => {
     let input = data.RoutingKey + " " + data.ComponentCode + " " + JSON.stringify(data.Event);
+    log.debug(input);
     return input;
 };
 
-export let convertCommandDataToWebsocketInputFormat = (data: CommandData): string => {
-    let input = data.Command + " " + data.JsonMessage;
+export let convertCommandDataToWebsocketInputFormat = (commandData: CommandData): string => {
+    let input = commandData.Command + " " + JSON.stringify(commandData.Data);
+    log.debug(input);
```

Comment:
In the log level library, is there a method called IsDebugEnabled ?
It will be interesting (regarding the performance) to create log strings only when required